### ISSUE TITLE
Error: Login failed for

### ISSUE THREAD

**Issue by Gabriel-Aragao** [arc298/instagram-scraper#848]
I`m getting the error "Login failed for every time. I`ve changed the account, created a new account, changed the IP, created a VM instance in the GCP to run Instagram scrapper, and updated the versions of Instagram API and simulated devices in constants.py. Nothing worked, anyone has a clue of what should i try? and if anyone Knows other tool to scrape Instagram comments could be useful too.

**Comment by Gabriel-Aragao** [comment]
KeyError: 'This error usually indicates that your IP address has been rate limited by Instagram. Try with a different VPN or proxy.

I§m getting this after trying to update UA_SOTIRES and CHROME_WIN_UA on constants.py file

**Comment by nns33213** [comment]
> I§m getting this after trying to update UA_SOTIRES and CHROME_WIN_UA on constants.py file

@Gabriel-Aragao yes, changing `STORIES_UA` is the reason why you get the error. It's not clear why this happens. Some people say that you need to try for a few days with the changed UA until it starts working. You can still change `CHROME_WIN_UA` and I found that it helps sometimes if you're trying to avoid the ban.

Scraping Instagram has become extremely difficult in recent weeks, to the point that even **manually** opening stories on the web app gets you permanently banned after a short time.

Please do let us know if you figure something out.
